README.md

```diff
@@ -6,6 +6,13 @@ A serverless form submission application built and deployed to AWS with the Serv
 
 ![Architecture Diagram](./.images/architecture.svg?raw=true)
 
+## Operations
```

Comment:
@ayoneeyee  can we create a table similar to what is seen in the bigmac repo?
Branch Status and Release columns. I like that you can easily see the build status of each integration branch and also create a PR directly from there. I like that we're explaining the process in this Operations block. But I would also propose renaming it and moving it close to the top if not the very top of the README. I'd call it something like Release ... Frankly we want to get away from "operations" handling releases

Reply:
Got it. I will fix and update code now. Thanks